composer.json

```diff
@@ -34,7 +34,9 @@
     ],
     "require-dev": {
         "doctrine/coding-standard": "^9.0.0",
+        "phpstan/extension-installer": "^1.1.0",
```

Comment:
Can we install the extension without the extension installer?

Reply:
Yes, done.